server/players.py

```diff
@@ -35,18 +35,21 @@ def __init__(
         permission_group: int = 0,
         lobby_connection: "LobbyConnection" = None
     ):
-        super().__init__(player_id, login)
+        self._faction = 0
```

Comment:
I wouldn't use fraction `0`, it doesn't looks like it's undefined. Maybe use `None`?

Reply:
Commit has no intention to change original behaviour.